lib/pynput/keyboard/_xorg.py

```diff
@@ -112,14 +112,25 @@ def _from_media(cls, name, **kwargs):
         """
         return cls._from_symbol('XF86_Audio' + name, **kwargs)
 
+    @classmethod
+    def _from_ISO(cls, name, **kwargs):
```

Comment:
The silent naming convention in this library is to use lower case letters for function names, so `_from_iso` would be more appropriate.

Reply:
Solved in https://github.com/moses-palmer/pynput/pull/401/commits/e90dc23e1fb541a9f9f3aa852c196f1fa86a9048